include/deal.II/base/parameter_handler.h

```diff
@@ -1478,17 +1487,6 @@ class ParameterHandler : public Subscriptor
    */
   std::vector<std::function<void (const std::string &)> > actions;
 
-  /**
-   * Mangle a string so that it doesn't contain any special characters or
-   * spaces.
-   */
-  static std::string mangle (const std::string &s);
```

Comment:
Yes, thanks for moving these. They were part of the class from back in a time when anonymous namespaces weren't available or didn't work well. They're obviously better placed in the .cc file.